pipeline/src/pipeline.rs

```diff
@@ -739,7 +739,7 @@ impl<T: FieldElement> Pipeline<T> {
         Ok(())
     }
 
-    fn stage(&self) -> Stage {
+    pub fn stage(&self) -> Stage {
```

Comment:
where's this used now?

Reply:
Oh, it was useful for debugging and I thought it's fair that that's public!